components/config/opts.rs

```diff
@@ -15,7 +15,7 @@ use servo_geometry::DeviceIndependentPixel;
 use servo_url::ServoUrl;
 
 /// Global flags for Servo, currently set on the command line.
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
```

Comment:
It's very confusing that `default_opts()` and `Opts::default()` are different.

So I think:
 - `Opts` should manually implement `Default` instead of deriving it
 - `default_opts()` should be removed in favor of `Opts::default()`

Reply:
Oof, you're right.